monitoring/ai-monitor/src/main.py

```
@@ -0,0 +1,423 @@
+        try:
+            synced = await bot.tree.sync()
+            logger.info("slash commands synced: %d", len(synced))
+        except Exception as e:
+            logger.warning("slash sync failed: %s", e)
+
+        # 이미 구동 중이면 재시작 방지
+        if getattr(bot, "_periodic_started", False):
+            return
+        bot._periodic_started = True
+
+        asyncio.create_task(poll_loop(settings, prom, analyzer, reporter, store))
```

Ruff (0.15.11)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)